### ISSUE TITLE
Add support for README.rst (reStructuredText)

### ISSUE THREAD

**Issue by vimalkvn** [docker/hub-feedback#621]
At the moment, a README.rst file in the Github repository is not displayed correctly on the Docker Hub web interface. Please check https://hub.docker.com/r/vimalkvn/docker-openlava-lsa/ for an example. Thanks!

**Comment by ajsmith** [comment]
+1 This would be very useful for those of us who use reStructuredText and Sphinx to build documentation.

**Comment by eggshell** [comment]
+1 would really love to see this on the site, since all of my documentation is built with reStructuredText in mind.

**Comment by kirecek** [comment]
+1

**Comment by Stibbons** [comment]
+1 Please add support for rst. It is so easy to do.

**Comment by Jitsusama** [comment]
I'm getting bit by this right now. Please don't make me switch to markdown :( :( :(

**Comment by Stibbons** [comment]
for the moment, i workaround this issue by generating 2 README using pandoc in my python project. Same pbl than wih Pypi, which does not support markdown correctly

**Comment by darrenchang** [comment]
Is there a plan to add support for rst on Docker Hub?

**Comment by Zethson** [comment]
Any update on this?

**Comment by sappelhoff** [comment]
This issue is still relevant, RST support for READMEs on Dockerhub would be highly appreciated.

**Comment by agarny** [comment]
Indeed, more than 4 years down the line and RST is still not supported... 😕

**Comment by xuhdev** [comment]
This has never been outdated and long due

**Comment by Zethson** [comment]
No

**Comment by domdfcoding** [comment]
IMO this is still a worthwhile (and important) feature
